has gulp-util dependency but no that plugin after installation

Error: Cannot find module 'gulp-util'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (D:\SVN\dev\trunk\src\3g\mobile-prototype\testGulp2\no
de_modules\gulp-scss\lib\scss.js:8:11)
    at Object.<anonymous> (D:\SVN\dev\trunk\src\3g\mobile-prototype\testGulp2\no
de_modules\gulp-scss\lib\scss.js:93:4)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)

BTW, I'm so disappointed that this plugin does not support compass.

--- a/package.json
+++ b/package.json
@@ -31,13 +31,13 @@
     "gulp-notify": "^2.2.0",
     "gulp-plumber": "^1.0.0",
     "gulp-sourcemaps": "^1.5.2",
-    "gulp-util": "^3.0.4",
     "mocha": "^2.2.4",
     "sinon": "^1.14.1"
   },
   "dependencies": {
     "convert-source-map": "^1.1.0",
     "q": "^1.3.0",
+    "gulp-util": "^3.0.4",
     "deepcopy": "^0.5.0",
     "merge": "^1.2.0",
     "mkdirp": "^0.5.0",

